### DISCUSSION TITLE
Http / Https configuratoion

### DISCUSSION THREAD

**Discussion by shpsyte** [TanStack/query#5158] [Q&A] [answered]
Hi all, 

React query is changing from HTTP to HTTPS 

How can I change it? 

just because we're running the API on localhost and I don't want to setup certificate cause it's a painful 
so, how can I change it ?

I'm using react 18  next13

**Comment by shpsyte** [comment, marked as answer]
Is not react-query

**Reply by louis-young** [reply]
TanStack Query is query/mutation function implementation agnostic. They are just functions that return promises. As you said, this is nothing to do with TanStack Query 🙂
